components/Funders.vue

```diff
@@ -1,20 +1,23 @@
 <template>
 	<div class="funders">
-		<h2>Funded by</h2>
-		<ul>
-			<li v-for="f in funders" :key="f.id">
-				<nuxt-link :to="f.href">
-					<img :src="f.logo" :alt="f.logoAlt">
-					{{ f.title }}
-				</nuxt-link>
-			</li>
-		</ul>
+		<p>Supported by Innovation, Science and Economic Development Canada’s Accessible Technology Program.</p>
+		<div>
+			<a href="https://www.ic.gc.ca/eic/site/118.nsf/eng/home">
+				<CanadaLogo />
+				<span class="screen-reader-only">Accessible Technology Program</span>
+			</a>
+		</div>
```

Comment:
This may be a question for @lliskovoi. Should the Paragraph be included in the link, or maybe the Program's name. The link is currently from the Canada logo, and also includes some AT only text. 

![Screen Shot 2020-04-01 at 11 17 34 AM](https://user-images.githubusercontent.com/556024/78154571-67e86580-740a-11ea-86d4-88832390e538.png)

Reply:
@cindyli the space between the logo and the text seems wider in the implementation than in the mockups. In the mockups they are 13px apart. Which would be about 0.8125 rems (if 16px base font size).

Implementation:
![Screen Shot 2020-04-01 at 11 19 40 AM](https://user-images.githubusercontent.com/556024/78154819-b564d280-740a-11ea-8b1c-6bf6162f793b.png)

Mockups:
![Screen Shot 2020-04-01 at 11 20 58 AM](https://user-images.githubusercontent.com/556024/78154997-ea712500-740a-11ea-939b-553d9569e7cc.png)

Reply:
@jobara Do you mean as a single focusable element? Would it look weird if only "Innovation, Science and Economic Development Canada's Accessible Technology Program" was included? Don't think we should include "Supported by" in the link text if we combine with the image. I would suggest either the logo only be a link, or just the "Innovation..." text.